tests/unit/test_controlTypes.py

```diff
@@ -10,6 +10,7 @@
 import controlTypes
 import versionInfo
 from . import PlaceholderNVDAObject
+from controlTypes.processAndLabelStates import _processNegativeStates, _processPositiveStates

```

Comment:
As much as this change is probably harmless, I don't see how it relates to the rest of the pr...

Reply:
This is a remaining deprecation that needed to be be performed. It was a bit verbose to add to the title and could be handled by a separate PR - it's justified in the PR description.